tests/execution/auth-jwt-const-env.md

```diff
@@ -0,0 +1,77 @@
+# Auth with JWT loaded from const env
+
+```graphql @server
+schema @server(port: 8000, graphiql: true, auth: [{jwt: {jwks: {data: "{{env.JWKS}}"}}, id: "jwt"}]) {
```

Comment:
Simply link the JWKS and HTPASSWD files and Tailcall should figure out what and how to authenticate.